### PR TITLE
OCPBUGS#175 - Change CodeReady product names to new names

### PR DESCRIPTION
**Pull request by SNiemann15**
Changed product names to OpenShift Locals (formerly CodeReady Containers) and OpenShift Dev Spaces (formerly CodeReady Workspaces) related to https://github.com/openshift/openshift-docs/pull/49363

Version(s): 4.11 +

Issue: https://issues.redhat.com/browse/OCPBUGS-175

Link to docs preview:
https://50074--docspreview.netlify.app/openshift-enterprise/latest/welcome/learn_more_about_openshift.html#Developer
https://50074--docspreview.netlify.app/openshift-enterprise/latest/welcome/oke_about.html

Additional information: 
* Related Release Notes PR: https://github.com/openshift/openshift-docs/pull/50035
* Applied changes to the odo files, which are currently not published. 
* Occurrences of CodeReady in the ROSA docs are not included.

### CODE REVIEW

File: welcome/learn_more_about_openshift.adoc [hunk 1]
```diff
@@ -118,7 +118,7 @@ Use the following sections to find content to help you learn about and use {prod
 | link:https://developers.redhat.com/[Red Hat Developers site]
 | xref:../cicd/builds/understanding-image-builds.adoc#understanding-image-builds[Builds]
 
-| link:https://developers.redhat.com/products/codeready-workspaces/overview[Red Hat CodeReady Workspaces]
+| link:https://developers.redhat.com/products/openshift-dev-spaces/overview[{openshift-dev-spaces-productname}]
```
Comment (max-cx) [inline welcome/learn_more_about_openshift.adoc:121]: The full product name is ` Red Hat OpenShift Dev Spaces`.

In which file do you keep the value for the `{openshift-dev-spaces-productname}` attribute?

File: welcome/oke_about.adoc [hunk 2]
```diff
@@ -299,8 +299,8 @@ s| Feature s| {oke} s| {product-title} s| Operator name
 | Red Hat OpenShift Pipelines Operator | Not Included | Included | OpenShift Pipelines Operator
 | Embedded Component of IBM Cloud Pak and RHT MW Bundles | Not Included | Included | N/A
 | Red Hat OpenShift GitOps | Not Included | Included | OpenShift GitOps
-| Red Hat CodeReady Workspaces | Not Included | Included | CodeReady Workspaces
-| Red Hat CodeReady Containers | Not Included | Included | N/A
+| {openshift-dev-spaces-productname} | Not Included | Included | {openshift-dev-spaces-productname}
```
Comment (max-cx) [inline welcome/oke_about.adoc:302]: I don't understand the meaning of this line and the `| Not Included | Included |` part.

**Comment by max-cx** [comment]
Looks like this image in https://50074--docspreview.netlify.app/openshift-enterprise/latest/welcome/oke_about.html needs to be updated regarding both _CodeReady Workspaces_ and _CodeReady Containers_:
image

File: welcome/learn_more_about_openshift.adoc (hunk 1 above)
Comment (max-cx) [inline welcome/learn_more_about_openshift.adoc:121]:
```suggestion
| link:https://developers.redhat.com/products/openshift-dev-spaces/overview[{openshift-dev-spaces-productname} (formerly Red Hat CodeReady Workspaces)]
```

**Comment by kalexand-rh** [comment]
If CM is successful, a separate QE review isn't required.

**Review by adellape — APPROVED** [review]

**Comment by pneedle-rh** [comment]
@SNiemann15 hi! The updated `images/oke-about.png` graph in this PR uses the plural "OpenShift Locals". Is this intentional or should the product name in the graph be "OpenShift Local"?

File: cli_reference/developer_cli_odo/creating_and_deploying_applications_with_odo/creating-a-multicomponent-application-with-odo.adoc [hunk 3]
```diff
@@ -13,7 +13,7 @@ This example describes how to deploy a multicomponent application - a shooter ga
 .Prerequisites
 
 * `{odo-title}` is installed.
-* You have a running cluster. Developers can use link:https://access.redhat.com/documentation/en-us/red_hat_codeready_containers/[CodeReady Containers (CRC)] to deploy a local cluster quickly.
+* You have a running cluster. Developers can use link:https://access.redhat.com/documentation/en-us/red_hat_openshift_local/[{openshift-local-productname}] to deploy a local cluster quickly.
```
Comment (pneedle-rh) [inline cli_reference/developer_cli_odo/creating_and_deploying_applications_with_odo/creating-a-multicomponent-application-with-odo.adoc:16]: It looks like the four CLI reference pages that are updated in this PR are not live in any of the topic maps. Given this, I can't review a preview of how `{openshift-local-productname}` would render in those cross-references if those pages were made live. But, given the use of the same attribute in `welcome/oke_about.adoc`, I think they look ok.

File: modules/codeready-workspaces.adoc [hunk 4]
```diff
@@ -4,15 +4,15 @@
 // * adding_service_cluster/rosa-available-services.adoc
 
 [id="codeready-workspaces_{context}"]
-= Red Hat CodeReady Workspaces
+= {openshift-dev-spaces-productname}
```
Comment (pneedle-rh) [inline modules/codeready-workspaces.adoc:7]: It looks like this module is not included in any assemblies in `main`. Given this, I can't review a preview of how the updated attributes would render if the module was made live. But, given the use of the same attributes in `welcome/oke_about.adoc`, I think they look ok.

**Comment by pneedle-rh** [comment]
LGTM

**Pull request merged**